practical/groovy/analyse_particles_for_another_user.groovy

```diff
@@ -62,17 +68,8 @@ import ij.plugin.frame.RoiManager
 import ij.measure.ResultsTable
 
 
-// Setup
-// =====
-
-// OMERO Server details
-HOST = "workshop.openmicroscopy.org"
-PORT = 4064
 group_id = -1
-//  parameters to edit
-dataset_id = 2331
-USERNAME = "username"
-PASSWORD = "password"
+
 // If you want to do analysis for someone else,
 // specify their username
 target_user = ""
```

Comment:
Yes, here is the crucial input for this script to work on others data - but this input is missing from the dialog.